ext/bg/js/backend.js

```diff
@@ -97,6 +98,10 @@ class Backend {
         }
 
         this.anki = options.anki.enable ? new AnkiConnect(options.anki.server) : new AnkiNull();
+
+        if (options.parsing.enableMecabParser) {
```

Comment:
Move into ```onOptionsUpdated```; there is also a related comment about how this is toggled on/off from the settings page.

Reply:
Are you sure you meant that this `if` block should be moved to `onOptionsUpdated`? It's in `applyOptions` that's called from `onOptionsUpdated` and other things in this block do similar things.

Reply:
You're right, sorry about that; I overlooked where it was called from after seeing the ```mecab.start/stopListener``` calls in the settings.js